### PR TITLE
improve user documentation regarding starting repairs

### PR DESCRIPTION
**Pull request by varjoranta**
* clean up given cluster name to match Reaper format

### CODE REVIEW

**Comment by Yarin78** [comment]
Looks fine, but build is not working?

**Comment by varjoranta** [comment]
Forgot to fix the unit tests after change. Luckily we have Travis :)

File: src/test/java/com/spotify/reaper/cassandra/JmxProxyTest.java [hunk 1]
```diff
@@ -21,7 +23,7 @@
 
   @Test
   public void testGetSymbolicName() {
```
Comment (Bj0rnen) [inline src/test/java/com/spotify/reaper/cassandra/JmxProxyTest.java]: We should move this to ClusterResourceTest or its own ClusterTest.

**Comment by Bj0rnen** [comment]
:+1: 

But I still think it might be a good idea to catch all exceptions, not just ReaperException, in RepairRunner.run(). Otherwise something like this might happen again.

**Comment by varjoranta** [comment]
I think there is still one more bug, but didn't have time to figure it out. Upstart logs actually catch the stderr, and did have the "unseen" exception.

**Comment by Bj0rnen** [comment]
I made a pull request, catching RuntimeException on top of ReaperException
(to log before the thread stops). I'll give the repair another shot.

On Sat, Jan 31, 2015 at 12:51 PM, Hannu Varjoranta <notifications@github.com

**Pull request merged**